File: ios/Classes/ChannelCreatedEvent.swift
/* Copyright Airship and Contributors */

import Foundation

struct ChannelCreatedEvent: TiEvent {
    let name: String = EventNames.channelCreated
    let data: [String: Any]
    init(channelID: String) {
        self.data = ["channelId": channelID]
    }
}

File: android/src/ti/airship/events/ChannelCreatedEvent.kt
/* Copyright Airship and Contributors */
package ti.airship.events

import ti.airship.TiAirshipModule

class ChannelCreatedEvent(channelId: String) : Event {
    override val data: Map<String, Any> = mapOf("channelId" to channelId)
    override val name: String = TiAirshipModule.eventChannelCreated
}